Source/WebInspectorUI/UserInterface/Views/ShaderProgramTreeElement.js

```diff
@@ -33,8 +33,7 @@ WI.ShaderProgramTreeElement = class ShaderProgramTreeElement extends WI.GeneralT
         super("shader-program", shaderProgram.displayName, subtitle, shaderProgram);
 
         // FIXME: add support for disabling/highlighting WebGPU shader pipelines.
```

Comment:
I wonder if there is a bug tracking this, or if this is the most aspirational type of `FIXME` (the type no one filed)?

Reply:
looks like there is <[https://webkit.org/b/201856]()>

not sure why it isn't linked here 🤔